lib/versioning.h

```diff
@@ -148,11 +151,10 @@
 #define VERSION_VALUE_3_24 0x0318
 #define VERSION_VALUE_3_25 0x0319
 
-
 /* config version code, in the same format as GlobalConfig->version */
-#define VERSION_VALUE   0x0319
-#define VERSION_CURRENT VERSION_3_25
-#define VERSION_CURRENT_VER_ONLY "3.25"
+#define VERSION_VALUE_CURRENT   0x0319
+#define VERSION_STR_CURRENT     "3.25"
+#define VERSION_PRODUCT_CURRENT VERSION_3_25
```

Comment:
The `VERSION_VALUE` and `VERSION_STR` conventions are okay, but the `VERSION_PRODUCT_CURRENT` I don't understand.
The previous names were a bit clearer: 
- we had version number as an integer(VERSION_VALUE),
- we had version number as a string (VER_ONLY)
- we had a version number as a string including the product name (VERSION_xx).

The only usage of `VERSION_PRODUCT_CURRENT` is in a debug message, where the product name and the current version is needed.
I would remove this macro (so we don't have to bump it's version on every release) and refactor the debug message.

Reply:
As discussed this can go in as it is.
We may need to refactor this anyway, due to an issue with but that's not yet decided.